src/protocolgame.cpp

```diff
@@ -2703,6 +2675,25 @@ void ProtocolGame::sendVIP(uint32_t guid, const std::string& name, const std::st
 	writeToOutputBuffer(msg);
 }
 
+void ProtocolGame::sendVIPEntries() {
+	const std::forward_list<VIPEntry>& vipEntries = IOLoginData::getVIPEntries(player->getAccount());
+
+	for (const VIPEntry& entry : vipEntries) {
+		VipStatus_t vipStatus;
```

Comment:
May as well assign `VIPSTATUS_ONLINE` to `vipStatus` here, and remove the `else` statement below.